test/test_decomp.py

```diff
@@ -545,6 +545,27 @@ def test_unsupported(t):
     xfail(
         "nn.functional.upsample_bilinear", "", dtypes=(torch.uint8,)
     ),  # off by one error
+    xfail(
+        "eye",
+        dtypes=(
+            torch.float8_e4m3fn,
+            torch.float8_e5m2,
+            torch.float8_e4m3fnuz,
+            torch.float8_e5m2fnuz,
+        ),
+    ),
+}
+
+quick_failures = {
+    xfail(
+        "eye",
+        dtypes=(
+            torch.float8_e4m3fn,
+            torch.float8_e5m2,
+            torch.float8_e4m3fnuz,
+            torch.float8_e5m2fnuz,
+        ),
+    ),
```

Comment:
Again, why should it fail?

Reply:
I have added support for where_cpu and where_cuda. These skips are no need any more.